### PR TITLE
Additional changelog updates for 4.1

### PR DESCRIPTION
**Pull request by parente**
Some work for #662

* Grammar consistency
* More details about how users trigger new features
* Screenshot of new location of cell toolbar triggers
* Couple more "other improvements"
* Links to PR and issue lists in the 4.1 milestone on GitHub

/cc @Carreau

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -9,36 +9,42 @@ For more detailed information, see `GitHub <https://github.com/jupyter/notebook>
...
-- Added multiple-cell selection (``Shift-Up/Down`` to extend selection), and actions on multiple-cells, including cut/copy/paste and execute.
+    .. image:: /_static/images/cell-toolbar-41.png
+
+- Added *Restart & Run All* to the *Kernel* menu. Users can also bind it to a keyboard shortcut on action ``restart-kernel-and-run-all-cells``.
+- Added multiple-cell selection. Users press ``Shift-Up/Down`` or ``Shift-K/J`` to extend selection in command mode. Various actions such as cut/copy/paste, execute, cell type conversion, and so on now operate on all selected cells.
 
   .. image:: /_static/images/multi-select-41.png
   
-- Added a command palette for executing Jupyter actions by name
+- Added a command palette for executing Jupyter actions by name. Users press ``Cmd/Ctrl-Shift-P`` or click the new command pallete icon on the toolbar.
```
Comment (willingc) [inline docs/source/changelog.rst]: typo in second mention of "palette"

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -9,36 +9,42 @@ For more detailed information, see `GitHub <https://github.com/jupyter/notebook>
...
 
   .. image:: /_static/images/command-palette-41.png
   
-- Added the find and replace dialog (in the Edit menu or ``F`` in command-mode).
+- Added a *Find and Replace* dialog to the *Edit*. Users can also press ``F`` in command mode to show the dialog.
```
Comment (willingc) [inline docs/source/changelog.rst]: Should read as ... *Edit* menu.

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -9,36 +9,42 @@ For more detailed information, see `GitHub <https://github.com/jupyter/notebook>
...
-UI Changes:
+- Moved the cell toolbar selector into the *View* menu. Added a button that triggers a "hint" animation to the main toolbar so users can find the new location. (Click here to see a `screencast <https://cloud.githubusercontent.com/assets/335567/10711889/59665a5a-7a3e-11e5-970f-86b89592880c.gif>`__ )
 
-- Moved the cell toolbar selector into the view menu `[screencast] <https://cloud.githubusercontent.com/assets/335567/10711889/59665a5a-7a3e-11e5-970f-86b89592880c.gif>`__.
-- Add Restart & Run All to Kernel menu. Can be added as keyboard shortcut for action ``restart-kernel-and-run-all-cells``.
-- Added multiple-cell selection (``Shift-Up/Down`` to extend selection), and actions on multiple-cells, including cut/copy/paste and execute.
+    .. image:: /_static/images/cell-toolbar-41.png
+
+- Added *Restart & Run All* to the *Kernel* menu. Users can also bind it to a keyboard shortcut on action ``restart-kernel-and-run-all-cells``.
+- Added multiple-cell selection. Users press ``Shift-Up/Down`` or ``Shift-K/J`` to extend selection in command mode. Various actions such as cut/copy/paste, execute, cell type conversion, and so on now operate on all selected cells.
```
Comment (willingc) [inline docs/source/changelog.rst]: "... and so on" feels a bit open ended. Perhaps reword to "... and cell type conversions apply to all selected cells.

**Comment by willingc** [comment]
@parente Nice changes :cake: A few minor suggestions/typos. Thanks!

**Comment by parente** [comment]
Thanks for the edits @willingc. I made the changes and rebased on master to fix the conflict.

**Comment by Carreau** [comment]
Thanks !

**Comment by Carreau** [comment]
(merging as soon as Travis go green)

**Pull request merged**